### PR TITLE
Update make.js

### PR DESCRIPTION
**Pull request by ghost**
Fixes `[DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.` in Windows.

### CODE REVIEW

**Comment by kevzdev** [comment]
Sorry, when I created the initial file I tested it on MacOS, no problems there.
I checked Windows and it gave these warnings.

**Comment by megaxlr** [comment]
Check again, You modified the paths of the `domains` and `hostfile`.

File: compilers/src/make.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const fs = require('fs')
 
-const domains = fs.readFileSync('../../domains', 'utf8').split('\n')
+const domains = fs.readFileSync('domains', 'utf8').split('\n')
 
-fs.appendFile('hostfile', '##COINHIVE DOMAINS\n')
+fs.appendFile('hostfile', '##COINHIVE DOMAINS\n', ()=>{})
 domains.forEach((domain) => {
-  fs.appendFile('../../hostfile', '0.0.0.0 ' + domain + '\n')
```
Comment (megaxlr) [inline compilers/src/make.js]: Re-add the directory movement.

File: compilers/src/make.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 const fs = require('fs')
 
-const domains = fs.readFileSync('../../domains', 'utf8').split('\n')
```
Comment (megaxlr) [inline compilers/src/make.js]: Re-add the directory movement.

File: compilers/src/make.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+const domains = fs.readFileSync('domains', 'utf8').split('\n')
 
-fs.appendFile('hostfile', '##COINHIVE DOMAINS\n')
+fs.appendFile('hostfile', '##COINHIVE DOMAINS\n', ()=>{})
```
Comment (megaxlr) [inline compilers/src/make.js]: Are you sure this won't accidentally write the header on the second line? Have you done checks?

**Review by ghost — CHANGES_REQUESTED** [review]

File: compilers/src/make.js (hunk 3 above)
Comment (kevzdev) [reply compilers/src/make.js]: Checked, it's a new line.

File: compilers/src/make.js (hunk 1 above)
Comment (kevzdev) [reply compilers/src/make.js]: Done

File: compilers/src/make.js (hunk 2 above)
Comment (kevzdev) [reply compilers/src/make.js]: Done

**Pull request closed without merging**